### ISSUE TITLE
B-spline registration with non uniform control point grid

### ISSUE THREAD

**Issue by zmani** [SuperElastix/SimpleElastix#275]
Hello,

I am trying to perform a Bspline registration with a non uniform grid spacing. Is it possible in SimpleElastix?

**Comment by kaspermarstal** [comment]
No, B-Spline registration is only supported on a uniform grid. You might want to take a look at the [SplineKernelTransform](http://elastix.isi.uu.nl/doxygen/classelastix_1_1SplineKernelTransform.html) if you need non-uniform grids.
